scanner/src/main/java/edu/ucr/cs/riple/scanner/Config.java

```diff
@@ -49,6 +50,7 @@ public interface Config {
 
   Serializer getSerializer();
 
+  @Nonnull
```

Comment:
Are we using a checker that looks at `@Nonnull`? Or the IDE? Wonder if it's worth the effort to get the scanner/annotator code to check with NullAway. Probably low priority right now, though, to be honest :)

(No action needed on this, just wondering)

Reply:
Currently we are not using any checkers to look at these annotations. However, I think this is a really good idea and will add it eventually. #21 

I added this annotation just to make sure I am explicitly mentioning this property and also IDE will give some basic hints if we violate it.

Reply:
Sounds good. Note we might want to remove them if/when we enroll this code in NullAway, given that non-null is the default. For now, though, they are definitely useful if the IDE is checking them, even in a less sound way.